dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -129,4 +129,40 @@ CWPKI2026W.useraction=Review the certificate authority URI. Review the keystore
 
 CWPKI2027E=CWPKI2027E: The {0} file path for the ACME service is null or empty. The path provided is '{1}'.
 CWPKI2027E.explanation=The file path was null or empty and cannot be used for the domain and account keys.
-CWPKI2027E.useraction=Provide a valid file path in the configuration.
\ No newline at end of file
+CWPKI2027E.useraction=Provide a valid file path in the configuration.
+
+CWPKI2028E=CWPKI2028E: The ACME service could not create a new session to connect to the ACME certificate authority at the {0} URI. The error is {1}.
+CWPKI2028E.explanation=The certificate authority could not be contacted and a signed certificate cannot be requested.
+CWPKI2028E.useraction=Review the error message for details on the failure. Review the configured certificate authority URI. Verify that the URI can be successfully accessed by the calling server. Verify that the calling server can receive a response from the certificate authority.
+
+CWPKI2029E=CWPKI2029E: The ACME service could not access the keystore at the {0} file path to find the {1} alias certificate. The error is {2}.
+CWPKI2029E.explanation=The keystore could not be accessed while checking for an existing certificate. The request to fetch a new certificate will not be completed because the keystore cannot be accessed.
+CWPKI2029E.useraction=Review the error message for details on the failure. Verify the file location is correct and the server has write file permissions.
+
+CWPKI2030E=CWPKI2030E: The ACME service could not install a certificate under the {0} alias into the {2} keystore. The error is {2}.
+CWPKI2030E.explanation=The ACME service received a new certificate from the certificate authority but the certificate cannot be installed locally.
+CWPKI2030E.useraction=Review the error message for details on the failure.
+
+CWPKI2031E=CWPKI2031E: The {0} certificate subject name in the default certificate with the {1} serial number is an invalid distinguished name. The error is {2}.
+CWPKI2031E.explanation=The certificate subject name must be formatted as a distinguished name as defined by RFC 2253. For example, a distinguished name used in an Ldap server.
```

Comment:
For example, a distinguished name used in an Ldap server.

I think I understand what you mean but need to make it a complete sentence to clarify

A distinguished name that is used in an LDAP server is an example of the valid format.
or

The certificate subject name must be formatted as a distinguished name as defined by RFC 2253, similar to a distinguished name that is used in an LDAP server.